packages/chat-headless-react/package.json

```diff
@@ -39,7 +39,7 @@
   "homepage": "https://github.com/yext/chat-headless#readme",
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.5",
-    "@yext/chat-headless": "^0.5.8",
+    "@yext/chat-headless": "^0.6.0",
```

Comment:
i'm surprised this pass the GH action tests considering this version is not publish yet

Reply:
I was surprised too lol but it seems to work